robotools/evotools/commands.py

```diff
@@ -234,9 +238,15 @@ def evo_aspirate(
         Overwrites the liquid class for this step (max 32 characters)
     tips : list
         Tip(s) that will be selected; use either a list with integers from 1 - 8 or with tip.T1 - tip.T8
+    arm : Union[Literal[0], Literal[1]]
+        Which LiHa to use, if more than one is available
     max_volume
         Maximum allowed dilutor volume.
     """
+    # update max_volume (if no value was given) according to the maximum dilutor volume stated at the top
+    if max_volume is np.nan:
```

Comment:
```suggestion
    if np.isnan(max_volume):
```

Comparisons with floats are very dangerous